src/main/kotlin/apply/domain/administrator/Administrator.kt

```diff
@@ -0,0 +1,46 @@
+package apply.domain.administrator
+
+import org.springframework.security.core.GrantedAuthority
+import org.springframework.security.core.authority.SimpleGrantedAuthority
+import org.springframework.security.core.userdetails.UserDetails
+import support.domain.BaseEntity
+import javax.persistence.Column
+import javax.persistence.Entity
+
+@Entity
+class Administrator(
+    @Column(unique = true, nullable = true)
+    private val name: String,
+
+    @Column(nullable = true)
+    private val password: String,
```

Comment:
아래 table은 둘 다 Not Null이네요.
여기서 빼주죠~

```suggestion
    @Column(unique = true)
    private val name: String,

    @Column
    private val password: String,
```